scilpy/image/volume_space_management.py

```diff
@@ -4,6 +4,41 @@
 from dipy.core.interpolation import trilinear_interpolate4d, \
     nearestneighbor_interpolate
 from dipy.io.stateful_tractogram import Origin, Space
+from dipy.segment.mask import bounding_box
+
+
+from scilpy.utils.util import voxel_to_world
+
+
+class WorldBoundingBox(object):
```

Comment:
We need to discuss this more. This class is used as a pickle in the `scil_crop_volume.py` script. Moving it here won't work, it must be a top of module to be picklable (https://docs.python.org/3/library/pickle.html).

And even, it will bug all pickled WorldBoundingBox kept by scilpy user (a pickle is bound to the module in which it was created). As such, the script parameters `--input_bbox` won't work with old boxes.

If we intend to deprecate those boxes, we should deprecate this pickle usage instead and rely on another saving method for the bounding box. It is legacy and really the worst way of doing this job.

Reply:
@arnaudbore @frheault @EmmaRenauld I would move this code back to the script, if we all agree. With a todo to change the way it works, remove pickling from the solution.

Reply:
Ok. @frheault , I will remove scil_crop_volume from files to give to people at the retreat. Seems more complicated than expected.

Reply:
@AlexVCaron Tests were passing, though. Do you have an example of pickle file that we could add to the tests?

Reply:
We do not test the input and output pickle, so we don't check this behavior. I'll prepare a little package for here and the retreat.

Reply:
[Here is some data](https://usherbrooke-my.sharepoint.com/:u:/g/personal/vala2004_usherbrooke_ca/EWlYflA-EUVOmWaE8CcBjSsBo5BI1xZq1X9gWN3d6Sn14Q?e=wEfvA2). You have the pickled bbox, created from the current master.

Loading the box does not work. However, contrary to what I thought, it is still possible to create and load boxes with the bbox class in the module. I still think we should change to another way than pickle, but we'll talk about it at the retreat.

Reply:
Thanks. I will try to play with this eventually.